src/transformers/__tests__/buildColumn.test.ts

```diff
@@ -20,3 +27,127 @@ test("throws error when new attribute collides with existing", () => {
     (e) => expect(e.message).toMatch(/name already in use/)
   );
 });
+
+test("throws error when expression uses unbound values", async () => {
+  try {
+    await uncheckedBuildColumn(
+      DATASET_A,
+      "E",
+      "child",
+      "AttributeNameThatDoesntExist + 1",
+      "any",
+      jsEvalExpression
+    );
+  } catch (e) {
+    expect(e.message).toMatch("AttributeNameThatDoesntExist");
+  }
+});
+
+test("using attribute name as formula creates clone of attribute", async () => {
+  expect(
+    (
+      await uncheckedBuildColumn(
+        DATASET_A,
+        "D",
+        "child",
+        "C",
+        "any",
+        jsEvalExpression
+      )
+    ).records.map((row) => row.D)
+  ).toEqual(DATASET_A.records.map((row) => row.C));
+});
+
+test("build column does nothing to dataset with no records", async () => {
+  const COLLECTION_A_OUTPUT = cloneDataSet(EMPTY_RECORDS);
+  COLLECTION_A_OUTPUT.collections[0].attrs?.push({
+    description:
+      "An attribute whose values were computed with the formula A + B * F + 1",
```

Comment:
Oops I forgot to change this--Can you wrap these formulas in backticks and also make that change in `buildColumn.ts` (line 90)? I've been trying to wrap them in backticks but I missed this one.

Reply:
Pushed in recent commit. Everything look good?

Reply:
Nice looks good!